src/qt/configuremasternodepage.h

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2018-2019 The ZENZO developers
+// Copyright (c) 2018-2019 The QuantisNet developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_QT_CONFIGUREMASTERNODEPAGE_H
+#define BITCOIN_QT_CONFIGUREMASTERNODEPAGE_H
+
+#include "masternodelist.h"
+#include "wallet/wallet.h"
+
+#include <QAbstractButton>
+#include <QAction>
+#include <QDialog>
+#include <QList>
+#include <QMenu>
+#include <string>
+#include <QPoint>
+#include <QString>
+#include <QTreeWidgetItem>
+
+class WalletModel;
+class MasternodeList;
+
+namespace Ui
+{
+class ConfigureMasternodePage;
+}
+
+QT_BEGIN_NAMESPACE
+class QDataWidgetMapper;
+QT_END_NAMESPACE
+
+/** Dialog for editing an address and associated information.
+ */
+class ConfigureMasternodePage : public QDialog
+{
+    Q_OBJECT
+
+public:
+    enum Mode {
+        NewConfigureMasternode,
+        EditConfigureMasternode
+    };
+
+    explicit ConfigureMasternodePage(Mode mode, QWidget* parent);
+    ~ConfigureMasternodePage();
+
+	void counter(int counter);
```

Comment:
nit,might be better if you use a beautify plugin to autoformat the h and cpp files.